lib/minitest/reporters/base_reporter.rb

```diff
@@ -64,8 +58,8 @@ def result(test)
       end
 
       def test_class(result)
-        if result.respond_to? :klass
-          result.klass
+        if result.respond_to?(:klass) && result.klass
+          Object.const_get(result.klass)
```

Comment:
Looks like this method is not defined in ruby 1.9.3.  Could you please fix this.